components/common/images/Logo.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import Image from 'next/image';
+import { useRouter } from 'next/navigation';
+import logo from './top_bar_logo.webp';
+
+const Logo = () => {
+  const router = useRouter();
```

Comment:
Instead of useRouter we can just use next/link

```
'use client';
import Image from 'next/image';
import NextLink from 'next/link';

export const Logo = () => {
  return (
    <NextLink href={'/'} passHref>
      <Image
        src="/images/top_bar_logo.webp"
        alt="Go to landing page"
        width={60}
        height={60}
      />
    </NextLink>
  );
};
```